### ISSUE TITLE
Please change "CTRL-C" for pause and resume

### ISSUE THREAD

**Issue by nathandragun** [madMAx43v3r/chia-plotter#467]
Please don't use SIGTERM to go into the pause/resume cycle, instead use "P" or something to pause and resume.

The current implementation basically forces you to wait until the current plot cycle is completed, regardless of your need to immediately kill the process.

It is impossible to kill the process with the current implementation even if you send a SIGKILL or SIGTERM via kill.

**Comment by kastermight** [comment]
`ctrl + z` to suspend the process, then `kill -9 <pid>` to actually kill it. You can grab process id from the suspension message.

**Comment by nathandragun** [comment]
Looks like this pull request might work for dealing with immediate terminations, but I don't think programmatically using CTRL-C like this is a good idea.

https://github.com/madMAx43v3r/chia-plotter/pull/475

**Comment by fermosan** [comment]
In a multi-platform world this is a simple and portable solution.  The purpose of the project is to create a plotter not a fully featured ecosystem to support full plotting management. The modification now provides all the previous functionality (gracefully exit) and provides immediate exit if needed. After all, what else one can do with ctrl-c ? 😅

**Comment by Mar-85** [comment]
Hold on, I was wondering why I needed to press ctrl-c a few times to quit. Are we saying here that pressing ctrl-c once will pause? How do we then resume?

**Comment by fermosan** [comment]
There never was a "pause" function.  

Previous Behavior: 
Ctrl-c once -> Gracefully exit ( the plotter will exit after completition and copy of current plot)
Ctrl-c Once -> Cancels the Gracefully exit and the plotter continues normally

Current Behaviorr:
Ctrl-c once -> Gracefully exit ( the plotter will exit after completition and copy of current plot)
Ctrl-c twice -> Exits instantly canceling the current ploting process

**Comment by madMAx43v3r** [comment]
"Current Behaviorr" is in PR #475

**Comment by nathandragun** [comment]
> There never was a "pause" function.
> 
> Previous Behavior:
> Ctrl-c once -> Gracefully exit ( the plotter will exit after completition and copy of current plot)
> Ctrl-c Once -> Cancels the Gracefully exit and the plotter continues normally
> 
> Current Behaviorr:
> Ctrl-c once -> Gracefully exit ( the plotter will exit after completition and copy of current plot)
> Ctrl-c twice -> Exits instantly canceling the current ploting process

Yeah, a poor choice of words on my part calling it a "pause" function.

**Comment by madMAx43v3r** [comment]
PR #475 is merged now, there is no more resume, just Gracefully exit or Exits instantly

**Comment by ByronAP** [comment]
I liked being able to cancel a cancel

**Comment by madMAx43v3r** [comment]
well, there is still a way to do that, at least on linux, just press key up and enter, after chia_plot is finished the terminal will run the same command again
